src/components/ThreeDNodeTree/stories/basic.story.mdx

```diff
@@ -0,0 +1,246 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import { ThreeDNodeTree } from '../ThreeDNodeTree';
+import { 
+  decorators,
+  onSelectTreeNode,
+  RightClickExample,
+  customStyle,
+  exampleTheme,
+  expandedKeys,
+} from './helper';
+import { ThemeProvider } from 'styled-components';
+
+<Meta title="Others|ThreeDNodeTree" decorators={decorators} />
+
+# ThreeDNode Tree
+
+Visualize parent-child relationship of 3D nodes in a tree structure. It will initially fetch all available root from a model. When a node is expanded, its children will be fetched dynamically. It behaves the same with Asset Tree.
+
+To retrieve a tree you need to provide:
+
+- `modelId` – you can find it using `sdk.ThreeD.listModels()` call
+- `revisionId` – you can find it via `sdk.ThreeD.listRevisions(modelID)` call
```

Comment:
this is outdated, please update it according to:
https://github.com/cognitedata/cognite-sdk-js/blob/v1/guides/MIGRATION_GUIDE_1xx_2xx.md

ask if not clear